src/components/cylc/GraphSubgraph.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <g v-if="subgraph.label!='margin'" class="c-graph-subgraph">
+    <rect
+      :width="subgraph.width"
+      :height="subgraph.height"
+      :x="subgraph.x"
+      :y="subgraph.y"
+      rx="50"
+      ry="50"
+      fill="none" />
+    <text
+      :x="calcLabelXPosition"
+      :y="calcLabelYPosition"
+      font-family="Roboto"
+      alignment-baseline="middle" text-anchor="middle"
+      font-size=70
+      fill="black"
+      stroke-width=1.5
+      stroke="white"
+    >
+      {{ subgraph.label }}
+    </text>
+  </g>
+</template>
+
+<script>
+export default {
+  name: 'GraphSubgraph',
+  props: {
+    subgraph: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    calcLabelXPosition () {
+      return (parseInt(this.subgraph.x) + (parseInt(this.subgraph.width) / 2))
+    },
+    calcLabelYPosition () {
+      return (parseInt(this.subgraph.y) + 90)
+    },
+  }
+}
+</script>
+
+<style lang="scss">
+  rect {
+    stroke-width: 8px;
+    stroke: grey;
+    stroke-dasharray: 50 50;
+  }
+</style>
```

Comment:
@MetRonnie, should this be `scoped`?

Or would it be easier to just inline this?

Reply:
Yes, either one of these should be done